cmd/kubeless/function.go

```diff
@@ -105,7 +61,28 @@ func getKV(input string) (string, string) {
 func readFile(file string) string {
 	data, err := ioutil.ReadFile(file)
 	if err != nil {
-		logrus.Fatalf("Can not read file: %s. The file may not exist", file)
+		logrus.Errorf("Unable to read file %s: %v", file, err)
```

Comment:
This will continue without the content of the file, potentially replacing an existing working Function with a broken one.